truffle/src/com.oracle.truffle.api/src/com/oracle/truffle/api/TruffleContext.java

```diff
@@ -757,6 +757,7 @@ public final class Builder {
         private Boolean allowPolyglotAccess;
         private Boolean allowEnvironmentAccess;
         private ZoneId timeZone;
+        private Consumer<String> onDeniedThreadAccess;
```

Comment:
please rename also private fields to threadAccessDeniedHandler and parameters everywhere:

Reply:
Renamed in 3d2506c